fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -2126,13 +2139,26 @@ private static void parseInclusionParameter(Class<?> resourceType, FHIRSearchCon
         String resourceTypeAndParameterName;
         String searchParameterTargetType;
 
-        SearchParameter searchParm;
-        InclusionParameter newInclusionParm;
-        List<InclusionParameter> newInclusionParms = null;
-
         List<String> allowedIncludes = getSearchPropertyRestrictions(resourceType.getSimpleName(), SEARCH_PROPERTY_TYPE_INCLUDE);
         List<String> allowedRevIncludes = getSearchPropertyRestrictions(resourceType.getSimpleName(), SEARCH_PROPERTY_TYPE_REVINCLUDE);
 
+        // Parse inclusionKeyword into parameter name and modifier (if present).
```

Comment:
Line 1904 is missing a double-quote, which messes up style-formatting in GitHub.

Reply:
Fixed.